scripts/gulpfiles/package_tasks.js

```diff
@@ -369,9 +397,18 @@ function packageReadme() {
  * referenced in package.json in the types property.
  */
 function packageDTS() {
-  return gulp.src(['typings/*.d.ts', 'typings/msg/*.d.ts'],
-                  {base: './typings'})
-    .pipe(gulp.dest(`${RELEASE_DIR}`));
+  const handwrittenSrcs = [
+    'typings/*.d.ts',
+    '!typings/blockly.d.ts',  // Exclude checked-in copy of blockly.d.ts.
+    'typings/msg/msg.d.ts',
+  ];
+  const builtSrcs = [
```

Comment:
Does this mean that there are actually two versions of typings being included?

Reply:
There are two sets of typings: handwritten ones, and generated ones.  The latter includes (only) `blockly.d.ts` and the various `msg/*.d.ts` files; all the rest are handwritten.

Previously they were all pulled from the same place—`typings/`—but now the generated ones are pulled from `build/` instead.  But there's still checked-in copies of the generated ones in `typings/`, and we want to exclude those (because they might be out-of-date if `npm run checkin:typings` hasn't been run recently) in favour of the generated ones.

So there are two versions, but only one is being included in the npm.